packages/mobile/patches/react-native-tab-view+3.1.1.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/node_modules/react-native-tab-view/src/SceneView.tsx b/node_modules/react-native-tab-view/src/SceneView.tsx
```

Comment:
this is so powerful. wow. very cool.

Is it possible to somehow reference in the patch file the fix PR (or maybe we keep a patches/README.md). Just so in the future if that PR does get merged / released, we know which patch we can get rid of. wdyt?

Reply:
yeah this is a great call! ill see if its part of their options, otherwise will add a readme for it